docs/tutorial/population_dynamics.rst

```diff
@@ -38,44 +39,89 @@ Gaussian Mixture Models
+The weights :math:`\phi_i` give the fraction of time spent in each state with the constraint :math:`\sum_i^K \phi_i = 1`. The means :math:`\mu_i` and
+standard deviations :math:`\sigma_i` indicate the average signal and noise for each state, respectively.
 
-    gmm = lk.GaussianMixtureModel.from_channel(force, n_states=2)
-    print(gmm.weights) # [0.53070846 0.46929154]
-    print(gmm.means)   # [10.72850757 12.22295543]
-    print(gmm.std)     # [0.21461742 0.21564235]
+The Pylake GMM implementation :class:`~lumicks.pylake.GaussianMixtureModel` is a wrapper around :class:`sklearn.mixture.GaussianMixture` with some
+added convenience methods and properties for working with C-Trap data. We can initialize a model from a force channel using :meth:`~lumicks.pylake.GaussianMixtureModel.from_channel`. Here we train a two-state model using the first 20 seconds of the force data to speed up the calculation::
 
-Here the force data is used to train a 2 state GMM. The weights give the fraction of time spent each state
-while the means and standard deviations indicate the average signal and noise for each state, respectively.
+    gmm = lk.GaussianMixtureModel.from_channel(force["0s":"20s"], n_states=2)
+
+We can inspect the parameters of the model with the :attr:`~lumicks.pylake.GaussianMixtureModel.weights`, :attr:`~lumicks.pylake.GaussianMixtureModel.means`,
+and :attr:`~lumicks.pylake.GaussianMixtureModel.std` properties. Note that, unlike the `scikit-learn` implementation, the states here are always ordered from
+smallest to largest value::
+
+    print(gmm.weights)  # [0.55505362 0.44494638]
+    print(gmm.means)  # [8.70803166 10.01637358]
+    print(gmm.std)  # [0.27888473 0.27492966]
 
 We can visually inspect the quality of the fitted model by plotting a histogram of the data overlaid with the weighted normal distribution probability density functions::
 
-    gmm.hist(force)
+    plt.figure()
+    gmm.hist(force["0s":"20s"])
+    plt.show()
 
-.. image:: pop_gmm_hist.png
+.. image:: figures/population_dynamics/gmm_hist.png
 
 We can also plot the time trace with each data point labeled with the most likely state::
 
-    gmm.plot(force['0s':'2s'])
+    plt.figure()
+    gmm.plot(force['0s':'1s'])
+    plt.show()
+
+.. image:: figures/population_dynamics/gmm_labeled_trace.png
+
+We can extract a list of dwell times (how long the system stays in one state before transitioning to another) using the
+:meth:`~lumicks.pylake.GaussianMixtureModel.extract_dwell_times` method::
 
-.. image:: pop_gmm_labeled_trace.png
+    dwell_times = gmm.extract_dwell_times(force)
+
+In the next section we'll fit these data to obtain lifetimes for the model states.
+
+.. note::
+
+    Note that we are not using the same slice of data that was used to train the model as the input for these methods;
+    once a model is trained, it can be used to infer the states from any data that is properly described by it.
```

Comment:
No, it actually is general. Any of the methods for this class can take other data; perhaps moving the note up a bit before the examples and some slight rewording would make it more clear?

Reply:
Yes, I suppose so. I wrote this comment, because you usually pick the same slice, `force["0s":"20s"]`, so unlikely the statement in this note, you often do use the same slice of data, eg for `gmm.hist`